pkg/solana/chainwriter/ata_creation.go

```diff
@@ -129,40 +129,48 @@ func (s *SolanaChainWriterService) handleATACreation(ctx context.Context, create
 		return fmt.Errorf("error enqueuing transaction: %w", err)
 	}
 
-	err = s.waitForTxFinality(ctx, ataUUID)
+	err = s.waitForTxStatus(ctx, ataUUID, types.Unconfirmed)
```

Comment:
Think a comment here mentioning we're waiting for confirmation might help since the common status type is a misnomer on the CW/TXM side.